frontend/src/admin/components/sidebar/userList/userListItem.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import {User} from "../../../models/user";
+
+interface UserListItemProps {
+    user: User,
+    onClick: () => void,
+    isSelected: boolean
+}
+
+export default function UserListItem(props: UserListItemProps) {
+    return (
+        <button
+            className={`${props.isSelected ? 'selected' : ''}`}
```

Comment:
You don't need to create a string template to make a ternary string selection.
```suggestion
            className={ props.isSelected ? 'selected' : '' }
```

Reply:
✅ oh yes, makes sense - this is now implemented